descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Edge search srs API",
+  "provides": [],
+  "requires": [
+    {
+      "id": "login",
+      "version": "5.0 6.0 7.0"
+    }
+  ],
+  "permissionSets": [],
+  "launchDescriptor": {
+    "dockerImage": "${artifactId}:${version}",
```

Comment:
Being a spring-based module, i gather that you need to fix maven replacement tokens in ModuleDescriptor [FOLSPRINGB-4](https://issues.folio.org/browse/FOLSPRINGB-4)

Reply:
This is still not fixed.